first.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>HTML page with links</title>
+<eta harset="utf-8">
+  <meta name="viewport cotent="width=device-width, initial-scale1">  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jqery.minjs">/script>
+      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></scipt>
+      /stye>
+      </head>
+      <body>
+
+<div class="jumbotron text-center">
```

Comment:
The code could be formatted a bit as it doesn't look good. 

You can try VS Code as an editor as it provides such capabilities

Reply:
I have reformatted the code using VS Code.